config/schemas/config.yaml

```
@@ -1358,6 +1358,612 @@ properties:
+                  default: false
+                  type: boolean
+                path:
+                  description: Path where the eBPF probe is located. It comes handy when the probe have been installed in the nodes using tools other than the init container deployed with the chart.
+                  type:
+                    - string
+                    - "null"
+                  $comment: Phase out the null default from the default config eventually
+              type: object
+      enabled:
+        default: true
+        type: boolean
+      falcoExporter:
+        additionalProperties: false
+        properties:
+          affinity:
+            $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+          nodeSelector:
+            $ref: '#/$defs/kubernetesNodeSelector'
+          resources:
+            $ref: '#/$defs/kubernetesResourceRequirements'
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+        type: object
+      falcoSidekick:
+        additionalProperties: false
+        properties:
+          affinity:
+            $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+          nodeSelector:
+            $ref: '#/$defs/kubernetesNodeSelector'
+          resources:
+            $ref: '#/$defs/kubernetesResourceRequirements'
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+        type: object
+      nodeSelector:
+        $ref: '#/$defs/kubernetesNodeSelector'
+      resources:
+        $ref: '#/$defs/kubernetesResourceRequirements'
+      rulesFiles:
+        additionalProperties: false
+        properties:
+          default:
+            additionalProperties: false
+            properties:
+              enabled:
+                default: true
+                type: boolean
+              version:
+                default: 2.0.0
+                type: string
+            type: object
+          incubating:
+            additionalProperties: false
+            properties:
+              enabled:
+                default: false
+                type: boolean
+              version:
+                default: 2.0.0
+                type: string
+            type: object
+          sandbox:
+            additionalProperties: false
+            properties:
+              enabled:
+                default: false
+                type: boolean
+              version:
+                default: 2.0.0
+                type: string
+            type: object
+        type: object
+      tolerations:
+        $ref: '#/$defs/kubernetesTolerations'
+      tty:
+        default: true
+        type: boolean
+    type: object
+  gatekeeper: true
+  kured:
+    additionalProperties: false
+    properties:
+      affinity:
+        $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+      configuration:
+        $comment: Would it be possible to extract schema from somewhere? Guess from the thing maybe?
```

```suggestion
        $comment: Would it be possible to extract schema from somewhere else in the future? 
```

Should we make this a little clearer so the next person working on it knows this is a thought left for future work.